bin/update_and_install_system_packages.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Could you add a quick line here that this script takes the list of parameters as input?